examples/fluids/densitycurrent.h

```diff
@@ -53,7 +53,9 @@ struct SetupContext_ {
   CeedScalar periodicity2;
   CeedScalar center[3];
   CeedScalar dc_axis[3];
+  CeedScalar wind[3];
```

Comment:
I know it's not relevant to this PR, but should these struct variables 
```
  CeedScalar periodicity0;
  CeedScalar periodicity1;
  CeedScalar periodicity2;
```
be removed now that we currently not support periodic BCs in this code? They are never used in neither `densitycurrent.h`, `advection2d.h` nor `advection.h`

Reply:
Yeah, we should delete unused variables like these. Periodicity will be supported at the mesh level, hopefully with no changes necessary in this example.

Reply:
Done. Thanks, @valeriabarra for catching it.